packages/web/components/navbar/index.tsx

```diff
@@ -318,10 +318,29 @@ export const NavBar: FunctionComponent<
               />
             </NotifiContextProvider>
           )}
+          <div className="group">
+            <a href="https://pro.osmosis.zone">
+              <Button
+                className="subtitle2 group mr-0 flex !w-40 transform items-center justify-center whitespace-nowrap bg-osmoverse-800 px-12 font-semibold tracking-wide text-osmoverse-200 transition-all duration-300 ease-in-out hover:px-6"
```

Comment:
because we want the text to slide to the side on hover and the icon to appear next to it, we need to force a constant button width otherwise the button will either grow or jump across the page.

Reply:
Should this be a code comment?